src/primitives.jl

```diff
@@ -179,6 +181,25 @@ end
 dropnull(X::NullableVector) = X.values[!X.isnull]    # -> Vector
 
 """
+    dropnull!(X::AbstractVector)
+
+Remove null entries of `X` in-place and return the updated vector. No-op
+if no nulls are present. If `X` is a `NullableVector`, a view of the
+unwrapped values is returned for convenience, although `X` itself will still
+be a `NullableVector`.
+"""
+function dropnull!(X::AbstractVector)
+	if !(Nullable <: eltype(X))
+		return X
```

Comment:
Good catch. I updated my text editor to use spaces instead of tabs so hopefully this won't happen again